services/elasticache/pkg/resource/cache_parameter_group/custom_api.go

```diff
@@ -133,6 +134,7 @@ func (rm *resourceManager) describeCacheParameters(
 			if awsErr, ok := ackerr.AWSError(respErr); ok && awsErr.Code() == "CacheParameterGroupNotFound" {
 				return nil, ackerr.NotFound
 			}
+			rm.log.Error(err, "Error during DescribeCacheParameters")
```

Comment:
Same comment about removing log.Error

Reply:
Please note that `err` is the wrong variable above. Should be `respErr`

Reply:
Updated code